lib/UrlQueryHelper.js

```diff
@@ -13,6 +13,10 @@ export default class UrlQueryHelper {
   }
```

Comment:
Changes in this file are unrelated, just ended up adding types to try something and kept them since it adds value.